src/electionguard/utils.py

```diff
@@ -1,11 +1,23 @@
 from datetime import datetime, timezone
+from enum import Enum
 from re import sub
 from typing import Callable, Optional, TypeVar
 from base64 import b16decode
 
 _T = TypeVar("_T")
 _U = TypeVar("_U")
 
+BYTE_ORDER = "big"
+BYTE_ENCODING = "utf-8"
+
+
+class ContestErrorType(Enum):
```

Comment:
Many of these were placed here due to circular dependencies due to the config in the serialize file.